lib/src/Uncertainty/Algorithm/Analytical/Analytical.cxx

```diff
@@ -111,8 +111,10 @@ void Analytical::run()
   /* set the level function of the algorithm */
   nearestPointAlgorithm_.setProblem(NearestPointProblem(standardEvent.getImplementation()->getFunction(), standardEvent.getThreshold()));
 
+    
   /* set the starting point of the algorithm in the standard space  */
-  nearestPointAlgorithm_.setStartingPoint(event_.getImplementation()->getAntecedent().getDistribution().getIsoProbabilisticTransformation().operator()(physicalStartingPoint_));
+  if ((nearestPointAlgorithm_.getImplementation()->getClassName() != "MultiStart"))
+    nearestPointAlgorithm_.setStartingPoint(event_.getImplementation()->getAntecedent().getDistribution().getIsoProbabilisticTransformation().operator()(physicalStartingPoint_));
```

Comment:
at least it should be a try/catch with NotDefinedException, this way pagmo algorithms could be used too

Reply:
I don't see how Pagmo can be directly used in FORM / SORM since Pagmo needs as first attribute the optimizationProblem to be solved.

 Do you have a script that could be shared to show an example of use of Pagmo in the context of FORM ?

Reply:
It is possible to initialize a Pagmo solver from the algo name, so it should look like this:
```
solver = ot.Pagmo("gaco")
solver.setStartingSample(x)
algo = ot.FORM(solver, event, x0)
```

Reply:
ok thanks

Reply:
okay. I tried your suggestion. But it seems that FORM problem is incompatible with Pagmo (maybe due to the equality constraint?). Pagmo returns an InvalidArgumentException : "No Pagmo support"

Reply:
try/catch is now implemented